packages/web-app-files/src/components/Search/List.vue

```diff
@@ -224,3 +326,8 @@ export default defineComponent({
   }
 })
 </script>
+<style lang="scss">
+.files-search-result .files-app-bar-actions {
+  display: none !important;
```

Comment:
you can just give the app-bar an `oc-hidden` class if I'm not wrong

Reply:
I only want to hide the actions, not the whole `app-bar` component.

Reply:
oh i see